source/huffman_generator/generator.c

```diff
@@ -321,8 +321,6 @@ int main(int argc, char *argv[]) {
         "\n"
         "#include <aws/compression/huffman.h>\n"
         "\n"
-        "#include <assert.h>\n"
```

Comment:
Not even code hidden in strings is safe from Justin's wrath 🔥

Reply:
it would have made clang-tidy mad immediately :P

Reply:
Let me tell you how much fun it is making sure generated code passes clang-tidy/format...

Reply:
`// clang-format off` ?

Reply:
Unfortunately, no, it's not for the includes, but most of the file.